components/icon/index.en-US.md

```diff
@@ -0,0 +1,125 @@
+---
+category: Components
+chinese: 图标
+type: Basic
+english: Icon
+toc: false
+---
+
+Meaningful vector graphics, each one can defeat an enemy.
+
+## The specification of naming icons
+
+We provide semantic name for every icon, and naming rules are as follows: 
+
+- Scanning line icon has the similar name with its solid one，but it's distinguished by `-o`, for example, `question-circle`(solid) and `question-circle-o`(scanning line);
```

Comment:
Can you explain a little bit what scanning line means?
I understand that `question-circle` show the question mark in a full circle and `question-circle-o` shows it an empty circle.